src/CMakeLists.txt

```diff
@@ -803,6 +803,19 @@ if(WITH_LIBCEPHFS)
   endif()
 endif(WITH_LIBCEPHFS)
 
+if(WITH_LIBCEPHSQLITE)
+  set(cephsqlite_srcs libcephsqlite.cc SimpleRADOSStriper.cc)
+  add_library(cephsqlite ${CEPH_SHARED} ${cephsqlite_srcs})
+  target_link_libraries(cephsqlite PRIVATE cls_lock_client librados ceph-common SQLite3::SQLite3 ${EXTRALIBS})
+  if(ENABLE_SHARED)
+    set_target_properties(cephsqlite PROPERTIES
+      OUTPUT_NAME cephsqlite
+      VERSION 1.0.0
+      SOVERSION 1)
```

Comment:
i'd suggest drop this `set_target_properties()` call. as cephsqlite is loaded using
```
.load libcephsqlite.so'
```
and we don't install multiple cephsqlite VFS with different so versions. and we don't need to change the `OUTPUT_NAME` as it is `cephsqlite` by default here as specified by the first argument of the `target_link_libraries()` call.

Reply:
Thanks, done.

Reply:
Kefu, I suspect the change I made in response to this comment caused libcephsqlite.so to not get built and/or packaged:

https://github.com/ceph/ceph/pull/39191#issuecomment-802101067

Do you see what I  did wrong?

Reply:
I see, I forgot to rename `debian/libcephsqlite-dev.install`.